Plugin~/WebRTCPlugin/VideoCaptureTrackSource.cpp

```diff
@@ -13,6 +13,10 @@
 
 namespace {
 
+    // Minimum interval is 10k fps.
+    #define FPS_TO_INTERVAL(fps) \
```

Comment:
Where is this used?

Reply:
it used by https://github.com/Unity-Technologies/com.unity.webrtc/blob/c673c01b29950cb08c11e8d2cb8c542ac6d4bd20/Plugin~/WebRTCPlugin/VideoCaptureTrackSource.cpp#L24 and https://github.com/Unity-Technologies/com.unity.webrtc/blob/c673c01b29950cb08c11e8d2cb8c542ac6d4bd20/Plugin~/WebRTCPlugin/VideoCaptureTrackSource.cpp#L28